### PR TITLE
meteor.js example

### LINKED ISSUE #35: Add Meteor example
Document how to install and run a Meteor app using a Dockerfile to install and run the app and connect to an external database.

### PR DESCRIPTION
**Pull request by andrei-markeev**
Fixes #35

So this is the latest Meteor.js at the time of writing, example itself is scaffolded using `meteor create --minimal` and slightly modified. Docker cloud v2 is used in `now.json`. Prerequisite is to have external MongoDb. I described those things in README.

One small ugly thing, is that I had to split the bundle, because `now` doesn't allow files more than 5Mb in the free tier, and the smallest Meteor.js bundle starts from 6Mb. Certainly I could unpack the bundle, but then it's like >10k files, uploading that much of files takes forever. I even created a [feature request](https://github.com/meteor/meteor-feature-requests/issues/324) regarding this to MDG.

### CODE REVIEW

File: node-meteor/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM mhart/alpine-node:8.11.4
+
+LABEL name="meteor-example"
+ARG HOSTNAME
+ARG NOW
+ARG NOW_DC
+ARG NOW_REGION
+ARG NOW_URL
```
Comment (TooTallNate) [inline node-meteor/Dockerfile]: These aren't necessary if they're not being consumed by the build process.

File: node-meteor/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM mhart/alpine-node:8.11.4
+
+LABEL name="meteor-example"
```
Comment (TooTallNate) [inline node-meteor/Dockerfile]: Is this one used for anything in the app code? If not we can remove it.

File: node-meteor/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN mkdir /app
```
Comment (TooTallNate) [inline node-meteor/Dockerfile]: Do `WORKDIR /app` here instead of mkdir, since `WORKDIR` creates the directory if it doesn't already exist in the image.

**Comment by TooTallNate** [comment]
> Error! No secret found by uid or name "mongodb-url"

I guess we need to add a demo mongodb secret to the account.

**Comment by andrei-markeev** [comment]
Yep, as I mentioned, external mongodb is a prerequisite. I tested both with Mongo Atlas and mLab - that is, with their free tiers that don't require a credit card. So hopefully setting this up won't be a problem!

**Comment by TooTallNate** [comment]
For the purposes of keeping the demo small and self-contained, I wonder if we could disable Mongo like in https://stackoverflow.com/a/18546761/376773.

**Comment by andrei-markeev** [comment]
Well, to be honest, this could be like super confusing for people out there. Meteor.js is a platform for building applications fast, hence I believe 99.99% of people are using Meteor with Mongo. Live data is one of cornerstones of Meteor.

I mean, first question when I try a Meteor deployment solution is always: "ok, now where do I put my MONGO_URL?". I haven't seen a single deployment solution for meteor (and there's a bunch) that doesn't have MONGO_URL. In worst case Mongo is deployed into same container, but then this is not really an option with `now` and it's immutable deployments.

I mean, if you want to make the example useful, then I would strongly recommend keeping MONGO_URL there. Is there a problem of getting a mongodb instance? After all, you can probably set up one in zeit if you don't want to use external dependencies?

**Comment by arunoda** [comment]
@TooTallNate, @andrei-markeev's point on MONGO_URL is a valid one.
We need to supply a MONGO_URL via a secret. (Since we deploy this app under zeit's account, we may need a different secret name)

@andrei-markeev here you've trying to build the bundle locally and then use that to deploy.
But what if we build the app inside ZEIT using a Dockerfile.

We can use [multi-stage](https://docs.docker.com/develop/develop-images/multistage-build/) to get the bundle and finally deploy it as a Node.js app. (As you've done here)
Then it won't have any Meteor related stuff inside it.

If we do ^^, we don't really need to worry about the bundle size anymore.

**Comment by andrei-markeev** [comment]
@arunoda good points!
I am quite new to Docker, never knew about multi-stage, thanks for the pointer 👍 

Did I understand correctly that you're thinking about something like this?
```docker
FROM meteor-tools
# build the bundle here

FROM alpine-node
COPY --from=0 /bundle .
EXPOSE 3000
CMD ["node", "main.js"]
```

The only concern I guess is how to handle all those possible different versions of meteor tool... But this is just an example after all so maybe we can go with something hardcoded. What do you think?

**Comment by arunoda** [comment]
> maybe we can go with something hardcoded

That's fine. This is just an example.

File: node-meteor/now.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "public": true,
+    "type": "docker",
+    "features": {
+        "cloud": "v2"
+    },
+    "env": {
+        "MONGO_URL": "@mongodb-url",
```
Comment (TooTallNate) [inline node-meteor/now.json]: Can you update the secret name to `@now-examples-mongodb-url` please?

**Comment by TooTallNate** [comment]
Wow what a great refactor! 👍

File: node-meteor/now.json (hunk 4 above)
Comment (andrei-markeev) [reply node-meteor/now.json]: Done!

**Comment by TooTallNate** [comment]
Yay it works 😄 https://node-meteor-ihslstytrl.zeit.sh/

**Comment by TooTallNate** [comment]
Thank you @andrei-markeev!

**Pull request merged**